NOTICE

```diff
@@ -0,0 +1,61 @@
+Apache Cassandra Cluster Manager
+Copyright 2024 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+
+This product originates, before git sha
+cc8cb98b29f68d12c8b5fa093a33dc3b346873b3, from software from DataStax
+and other individual contributors.
+
+Non-DataStax contributors are listed below. Those marked with
+asterisk have agreed to donate (copyright assign) their contributions to the
+Apache Software Foundation, signing CLAs when appropriate.
+
+Kishan Karunaratne <kishan@karu.io>
+Lucas Meneghel @lmr
+Joaquin Casares @joaquincasares
+Andrew Hust @nutbunnies
+Jaume Marhuenda <jaumemarhuenda@gmail.com>
+Daniel Compton <desk+github@danielcompton.net> *
+??? @steveandwang
```

Comment:
@steveandwang ,  may I get your full name to add here ?  (also your consent to donate your contributions to the ASF, as we move this code repository into the Apache Cassandra project, ref: https://github.com/riptano/ccm/issues/773 )